src/Tool-Registry/PharoCommonTools.class.st

```diff
@@ -104,7 +104,8 @@ PharoCommonTools class >> settingsOn: aBuilder [
 
 { #category : #'system startup' }
 PharoCommonTools class >> shutDown: aboutToQuit [
-	self allInstancesDo: #cleanUp
+	Smalltalk image tools cleanUp.
+	Smalltalk image resetTools
```

Comment:
I am curious when #image was introduced and why? I see that it is same as Smalltalk:
```Smalltalk
Smalltalk image == Smalltalk "==true"
```

Reply:
I think the idea was to split that eventually.

I would actually like to add a reserved variable "thisImage" to talk to the image.